contracts/svc/invocable-scripts.yml

```diff
@@ -0,0 +1,317 @@
+          type: string
+        op:
+          readOnly: true
+          description: op describes the logical code operation during error. Useful for debugging.
+          type: string
+        err:
+          readOnly: true
+          description: err is a stack of errors that occurred during processing of the request. Useful for debugging.
+          type: string
+      required:
+        - code
+        - message
+    Script:
+      properties:
+        id:
+          readOnly: true
+          type: string
+        name:
+          type: string
+        description:
+          type: string
+        orgID:
+          type: string
+        script:
+          description: script to be executed
+          type: string
+        language:
+          $ref: '#/components/schemas/ScriptLanguage'
+        url:
+          type: string
+          description: invocation endpoint address
+        createdAt:
+          type: string
+          format: date-time
+          readOnly: true
+        updatedAt:
+          type: string
+          format: date-time
+          readOnly: true
+      required:
+        - name
+        - orgID
+        - script
+    Scripts:
+      type: object
+      properties:
+        scripts:
+          type: array
+          items:
+            $ref: '#/components/schemas/Script'
+    ScriptCreateRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        orgID:
+          type: string
+        script:
+          description: script to be executed
+          type: string
+        language:
+          $ref: '#/components/schemas/ScriptLanguage'
+      required:
+        - name
+        - orgID
+        - script
+        - language
+    ScriptUpdateRequest:
+      type: object
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        script:
+          description: script is script to be executed
+          type: string
+    ScriptHTTPResponseData:
+      description: The data sent to end user when a script is invoked using http. User defined and dynamic
+      type: object
+    ScriptHTTPResponse:
+      description: The full response sent to end user when a script is invoked using http
+      allOf:
+        - $ref: '#/components/schemas/ScriptHTTPResponseNoData'
+        - type: object
+          properties:
+            data:
+              $ref: '#/components/schemas/ScriptHTTPResponseData'
+    ScriptHTTPResponseNoData:
+      description: The full response sent to end user when a script is invoked using http
+      properties:
+        type:
+          type: string
+          enum:
+            - http
+        dataType:
+          type: string
+          enum:
+            - json
+        headers:
+          type: object
+        statusCode:
+          type: string
+          enum:
+            - 200
+            - 500
+            - 404
```

Comment:
Is this necessary? It's not referenced anywhere in the rest of the document.

Reply:
`ScriptHTTPResponseData` is used here: https://github.com/influxdata/openapi/blob/371a8ca7e185dfeeafd354a3e7291de695d86d3b/contracts/svc/invocable-scripts.yml#L164

but I do not think we have a use for `ScriptHTTPResponseNoData`